front_end/README.md

```diff
@@ -0,0 +1,29 @@
+# front_end
+
+## Project setup
+```
+npm install
+```
+
+### Compiles and hot-reloads for development
+```
+npm run serve
+```
+
+### Compiles and minifies for production
+```
+npm run build
```

Comment:
Just for consistency with most of our other libraries you might update this to `npm run compile`